data/components.json

```diff
@@ -0,0 +1,850 @@
+[
+    {
+        "component_id":"acid_return",
+        "component_name":"Long-term fate of acid",
+        "quantification_target":"long-term fate of acid",
+        "uncertainty_type":"execution",
+        "responsibility":"project",
+        "uncertainty_impact_min":"low",
+        "uncertainty_impact_max":"low",
+        "description":"Any source of acid from the process that ultimately reaches the ocean or interacts with terrestrial carbonates prior to being neutralized will counteract claimed alkalinity enhancement. At the project level, it is possible to ascertain the likely end fate of produced acid using operational records. As this solution scales, acid disposal may become more of a challenge.",
+        "revisions":[
+            {
+                "date":"09-19-2022",
+                "note":"First release."
+            }
+        ],
+        "notes":"",
+        "pathways":[
+            "ocean-alkalinity-enhancement-electrochemical"
+        ]
+    },
+    {
+        "component_id":"alk_convert",
+        "component_name":"Mineral dissolution",
+        "quantification_target":"theoretical atmospheric removal",
+        "uncertainty_type":"execution",
```

Comment:
nit: possible to parse this into an array like how we do at the pathway level? I don't think we'll actually surface this on the web so not strictly necessary, but good for consistency if we're going to leave the field here.